substrate/frame/support/src/traits/tokens/fungible/item_of.rs

```diff
@@ -387,8 +387,8 @@ impl<
 		Imbalance::new(<F as fungibles::Balanced<AccountId>>::issue(A::get(), amount).peek())
 	}
 	fn pair(amount: Self::Balance) -> (Debt<AccountId, Self>, Credit<AccountId, Self>) {
-		let (a, b) = <F as fungibles::Balanced<AccountId>>::pair(A::get(), amount);
-		(Imbalance::new(a.peek()), Imbalance::new(b.peek()))
+		let (d, c) = <F as fungibles::Balanced<AccountId>>::pair(A::get(), amount);
+		(Imbalance::new(d.peek()), Imbalance::new(c.peek()))
```

Comment:
may be better to now cause a diff/change, where it's not really needed.
and I actually would prefer `a, b` in the context where you really care only about right order, receive order == return order.